PullRequestGuide.md

EthanFrisell opened the pull request:
## Using this Pull Request Template
* In the Description section below, please fill out the details of each of the bulleted items
* In the Type of Change section, check the box for all that apply. To check the box in markdown format, simply replace the blank space between the [ ] with an x (i.e., [x])
* In the Checklist Before Requesting a Review section, make sure you complete the required tasks that are pertinent to your task. Not all check boxes will need to be filled out for every type of task. Check the boxes as outlined previously (i.e., [x]).

## Description
Please document the following about this pull request:
* If this work is associated with a university course, please identify the university/course/semester (ex. MTU/MIS3200/Fall2023): MTU/MIS3200/Spring 2024
* List the team you are on: Education-System
* Identify the task/story this work pertains to: Adding a page.md to demonstrate how to utilize Pull Requests
* Describe what changes you made to the code/documentation: Making another page.

## Type of Change (check all that apply)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation addition/update (non-breaking change that adds documentation to the system)

## Checklist Before Requesting a Review
- [ ] I have followed the git practices outlined [here](GitPractices.md) to ensure code conflicts are minimized
- [ ] My code follows the style guidelines of this project outlined [here](documentation/StyleGuide.md)
- [ ] I have performed a self-review of my code. It runs without errors and follows the [Best Coding Practices](BestCodingPractices.md)
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] There are no API keys, passwords, usernames, or other secrets in the code. I am using .env files for all secrets.
- [ ] I have added tests that prove my fix is effective or that my feature works

EthanFrisell committed:
PullRequestGuide.md

jeffreywallphd reviewed (changes requested):
Please add a little content to the file before we merge it into the repository. Right now, the file is empty.

EthanFrisell committed:
Ongoing progress of PullRequestGuide.md

In this committed change, I did as asked and got started on the PullRequestGuide. I will continue to add to this guide to enable students to have an easier time making a Pull Request in GitHub.

EthanFrisell commented:
I have added to the empty file now and it should have the foundation for the PullRequestGuide. I hope I did the commit correctly, I am still getting used to GitHub. I will continue to work on the PullRequestGuide in my free time.

jeffreywallphd reviewed (approved):
Excellent work! This is a great start!!
